lib/cartopy/tests/mpl/test_path.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Crown and Cartopy Contributors
```

Comment:
I used `git mv` to change this from `test_patch.py` so I'm not clear why it's showing as a deletion and new file :confused:

Reply:
I moved the rename to its own commit, so it's now easier to see what I actually changed.  It also restored the blame that had gone missing, so I propose to keep that separate commit at merge.